articles/azure/azs-how-create-vmfromsnapshot-portal.md

```diff
@@ -0,0 +1,147 @@
+---
+title: How to create a virtual machine from a disk snapshot using the UKCloud Azure Stack Hub portal
+description: Provides help for creating a virtual machine from a managed disk snapshot using the portal on UKCloud for Microsoft Azure
```

Comment:
I'm not sure if we call it the `UKCloud Azure Stack Hub portal` anymore?

Reply:
This seems to be common wording in other docs.